src/ubuntu/22.04/cross/arm64/Dockerfile

```diff
@@ -4,13 +4,13 @@ FROM mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-22.04-crossdeps-local
 RUN apt-get update \
     && apt-get install -y \
         binutils-aarch64-linux-gnu \
-        nodejs \
     && rm -rf /var/lib/apt/lists/*
 
 ADD rootfs.arm64.tar crossrootfs
 
-# update node
-ENV NODE_VERSION 20.9.0
+# install node
 RUN cd ~ && \
-    curl -LO https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-linux-arm64.tar.xz && \
-    tar xf node-v${NODE_VERSION}-linux-arm64.tar.xz
\ No newline at end of file
+    curl -sL https://deb.nodesource.com/setup_20.x -o nodesource_setup.sh && \
+    sudo bash nodesource_setup.sh && \
+    sudo apt-get install nodejs -y && \
+    rm -rf /var/lib/apt/lists/*
```

Comment:
Should we also rm the `nodesource_setup.sh` or doesn't it matter?

Reply:
@mthalman what do you think?

Reply:
Might as well. It won't be needed in the final image. I'm a fan of image cleanliness.

Reply:
Done